GDJS/GDJS/Extensions/Builtin/CommonInstructionsExtension.cpp

```diff
@@ -347,11 +347,18 @@ CommonInstructionsExtension::CommonInstructionsExtension() {
       [](gd::BaseEvent& event_,
          gd::EventsCodeGenerator& codeGenerator,
          gd::EventsCodeGenerationContext& parentContext) {
-        gd::String outputCode;
         gd::WhileEvent& event = dynamic_cast<gd::WhileEvent&>(event_);
 
-        // Context is "reset" each time the event is repeated (i.e. objects are
-        // picked again)
+        // Prevent code generation if there are no conditions, as this would
+        // get the game stuck in a never ending loop.
+        if (event.GetWhileConditions().empty())
```

Comment:
I would also check for the conditions and actions inside the while, if absolutely everything is empty.

The idea is to avoid bug reports because "while does not work". It should just do nothing when there is nothing at all.